spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java

```diff
@@ -268,23 +270,11 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
 			bean = advised;
 		}
 
-		if (bean instanceof MessageChannel) {
-			DirectChannelMetrics monitor;
-			MessageChannel target = (MessageChannel) extractTarget(bean);
-			if (bean instanceof PollableChannel) {
-				if (target instanceof QueueChannel) {
-					monitor = new QueueChannelMetrics((QueueChannel) target, beanName);
-				}
-				else {
-					monitor = new PollableChannelMetrics(target, beanName);
-				}
-			}
-			else {
-				monitor = new DirectChannelMetrics(target, beanName);
-			}
-			Object advised = applyChannelInterceptor(bean, monitor, beanClassLoader);
+		if (bean instanceof MessageChannel && bean instanceof MessageChannelMetrics
+				&& bean instanceof NamedComponent) {
+			MessageChannelMetrics monitor = (MessageChannelMetrics) extractTarget(bean);
+			monitor.enableStats(true);//TODO: INT-3638
```

Comment:
Right - see the (parent) JIRA: 

- add defaultStatsEnabled and/or statsEnabledPattern to the IMBE to set the initial value for all/matching component bean names